Formula/open-mpi.rb

```diff
@@ -1,8 +1,26 @@
 class OpenMpi < Formula
   desc "High performance message passing library"
   homepage "https://www.open-mpi.org/"
-  url "https://www.open-mpi.org/software/ompi/v3.1/downloads/openmpi-3.1.0.tar.bz2"
-  sha256 "b25c044124cc859c0b4e6e825574f9439a51683af1950f6acda1951f5ccdf06c"
+
+  stable do
+    # These can be put back up under `homepage` once Open MPI v3.1.1 is released:
+    url "https://www.open-mpi.org/software/ompi/v3.1/downloads/openmpi-3.1.0.tar.bz2"
+    sha256 "b25c044124cc859c0b4e6e825574f9439a51683af1950f6acda1951f5ccdf06c"
+
+    # Fix https://github.com/Homebrew/homebrew-core/issues/26009 and its upstream counterpart
+    # https://github.com/open-mpi/ompi/issues/5000 for the time being.
+    #
+    # (Remove this patch when Open MPI v3.1.1 is shipped upstream, as it will be included in that
+    # release.)
+    #
+    # From the result of https://github.com/open-mpi/ompi/pull/5015 and
+    # https://github.com/open-mpi/ompi/pull/5160 as backported to Open MPI v3.1.x by
+    # https://github.com/open-mpi/ompi/pull/5118:
```

Comment:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Do you think this comment is too long?  How would you prefer it be worded?